File: router.py
#!/usr/bin/python

import sys, subprocess

classpath = ":".join([
    "jars/plume.jar",
    "jars/lib.jar"
])
classname = "edu.washington.cs.cse490h.lib.Router"

subprocess.call(["java", "-cp", classpath, classname] + sys.argv[1:])

File: router.pl
#!/usr/bin/perl

# Simple script to start a Router

main("edu.washington.cs.cse490h.lib.Router");

sub main {
    my $classname = shift;
    my $classpath = join ":", (
        "jars/plume.jar",
        "jars/lib.jar"
    );
    my $args = join " ", @ARGV;

    exec("java -cp $classpath $classname $args");
}
